motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -133,6 +134,9 @@ typedef enum
 	ROS_CMD_CHECK_MOTION_READY = 200101,
 	ROS_CMD_CHECK_QUEUE_CNT = 200102,
 	ROS_CMD_STOP_MOTION = 200111,
+	ROS_CMD_START_SERVOS = 200112, // starts the servo motors

```

Comment:
@shaun-edwards 
I know these command id's are motoman specific... but I wonder how the original number scheme was chosen.  Do these correspond to some similar numbering scheme in ROS-I SimpleMessage?
Are these chosen ID's ok?